plan/task.go

```diff
@@ -142,7 +142,16 @@ func (p *PhysicalIndexJoin) attach2Task(tasks ...task) task {
 }
 
 func (p *PhysicalIndexJoin) getCost(lCnt float64) float64 {
-	cst := lCnt * netWorkStartFactor
+	if lCnt <= 1 {
+		lCnt = 1
+	}
+	cst := lCnt * netWorkFactor
+	batchSize := p.ctx.GetSessionVars().IndexLookupSize
```

Comment:
Store it as a variable of `PhysicalIndexJoin`? This is also used when build IndexJoin Executor.